src/plone/restapi/pas/plugin.py

```diff
@@ -119,6 +119,9 @@ def authenticateCredentials(self, credentials):
 
         userid = payload['sub']
 
+        if isinstance(userid, unicode):
```

Comment:
I would omit that check for unicode as the decoded token always contains unicode strings. Or can you provide a test for the code path where the if statement evaluates to false?

Reply:
@Gagaro ^^^ :)

Reply:
I answered but it was stuck on pending, not sure what github changed this time..

`payload` comes from a method which could be overwritten. I wanted to be sure that I'm not trying to encode an already encoded string.

Reply:
@Gagaro `payload` comes from a private (underscore prefixed) method `_decode_token()`. It's pretty safe to assume that this method doesn't get overwritten. Even if the method changes in the future and for some reason returns byte strings then tests will fail. I prefer to have those kind of checks in tests instead of having an unreachable code path. Just my two cents.

Reply:
All right then, I removed the check.